ansible_collections/arista/avd/molecule/evpn_underlay_ebgp_overlay_ebgp/inventory/group_vars/DC1_FABRIC.yml

```diff
@@ -27,10 +27,12 @@ vxlan_vlan_aware_bundles: true
 # bgp peer groups passwords
 bgp_peer_groups:
   IPv4_UNDERLAY_PEERS:
+    name: UNDERLAY_PEERS
```

Comment:
Minor detail, but I think we should use "UNDERLAY*-*PEERS" in the tests. all other names we have are with "-" where variables are with "_". Same for the others.

Reply:
Okay, it's changed. Wanted to ask a question on whether or not is it best for me to add new commits with requested changes or just rebase requested changes into existing commits and then just force push the branch? Whatever is easier to review, doesn't give me any struggle to rebase it into nicer looking commits.
Also two more questions, I have a bunch of changes lined up for eos_designs, is it best if I create an issue before I put them into PRs and how do you also feel about taking over PRs that seem to be abandoned?

Reply:
Re. requested changes please just add new commits. We will squash merge if the history becomes too messy.
Re. please submit an issue. It is fine if you put related changed into the same issue. That will allow us to have a discussion before you spend too much time on a PR.
Re. Some of the abandoned PRs may not be relevant any longer, or we have replaced them and forgot to clean up. Please let us know which ones you are considering? (Comment with mention on the PRs thanks)